LibreHardwareMonitor/UI/AuthForm.cs

```diff
@@ -5,8 +5,11 @@
 // All Rights Reserved.
 
 using System;
+using System.Runtime.Remoting.Channels;
 using System.Windows.Forms;
 
```

Comment:
Just going to comment here, this applies to almost all modified files. Can you follow the code style / reformat document? Some files have empty lines now between using statements, double empty lines, etc.

Reply:
I will look into it all. Thanks